contributor-docs/code-contributions.md

```diff
@@ -243,6 +243,30 @@ The parameter `-Dtest=` can be given to test a single class (e.g., `-Dtest=Pubsu
 
 The same happens when the test is executed from an IDE, just make sure to add the parameters `-Dproject=`, `-DartifactBucket=` and `-Dregion=` as program or VM arguments.
 
+### Running Load Tests
+
+For information on adding and running load tests, see [Adding a Load Test](./add-load-test.md).
+
+### Adding New Templates
+
+If you are interested in introducing a new template, please file an issue using the [Google Issue Tracker](https://issuetracker.google.com/issues/new?component=187168&template=0) before doing so. You need approval before moving forward. Any new templates must be flex templates in the v2 directory.
+
+Once you have approval, follow these steps from the root of the repo to get started on your template.
+
+First, install the [maven plugin](#templates-plugin), then create a subdirectory in the repository: `v2/<name-of-your-template>`. `<name-of-your-template>` should follow the pattern `<source>-to-<sink>`. From there, you can follow the steps in [Adding a Flex Template](./add-flex-template.md) to develop your template.
+
+All new templates must comply with the following guidance:
+
+- [ ] Template addition has been approved by the core Dataflow Templates team.
```

Comment:
Not sure we have a contact information to reach our team. Maybe also open a github issue to track this new template job as well?